src/focusstack.h

```diff
@@ -0,0 +1,40 @@
+/***************************************************************************
+  focusstack.h - %{Cpp:License:ClassName}
+
+ ---------------------
+ begin                : 11.4.2018
+ copyright            : (C) 2018 by david
+ email                : david at opengis.ch
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+#ifndef FOCUSSTACK_H
+#define FOCUSSTACK_H
+
+#include <QList>
+#include <QtQuick/QQuickItem>
+
+class FocusStack : public QObject, public QList<QQuickItem*>
```

Comment:
I would probably make the `QList` a member rather than inheriting from it. With inheritance it's possible to directly interface with the QList from outside (adding/removing/reordering items etc.) while we really want all calls to go through methods in the `FocusStack` class. That it manages the stack internally as a QList is more of an implementation detail than public API.

Reply:
Hm... I thought about that as well. But I decided to inherit from it because it's anyway only about this stack list. But it's no big deal to change.

Reply:
Please do, I think it's cleaner (e.g. it leaves room for changing to a QVector if we think that's better [like some sources suggest](https://stackoverflow.com/a/43677426) ).

Reply:
Aight.